src/test/java/example/Example.java

```diff
@@ -5,6 +5,7 @@
 package example;
```

Comment:
But it lives in the test, why not use same package as the src package? so you don't need to all the imports from the src and it is clearer that this example app is for the src package. 

or you want to be clear that this is an independent app?